index.js

```diff
@@ -11,6 +11,17 @@ const itemForm = document.getElementById('itemForm');
 const itemsDiv = document.getElementById('itemsDiv');
 const priceAmount = document.getElementById('priceAmount');
 const clearFormButton = document.getElementById('cancelButton');
+// form inputs and err msgs
+const nameInput = document.getElementById('nameInput');
+const nameError = document.getElementById('nameError');
+const priceInput = document.getElementById('priceInput');
+const priceError = document.getElementById('priceError');
+// show/hide filter ui
+let isItemsControlHidden = false;
+const showItemsControlContainer = document.getElementById('showItemsControl');
+const itemsControl = document.getElementById('itemsControl');
+const toggleMessage = document.getElementById('toggleMessage');
+const toggleArrow = document.getElementById('arrow');
```

Comment:
This is all getting a bit long - next PR should focus on splitting up the JS into sensical modules